posthog/queries/trends/sql.py

```diff
@@ -60,7 +60,14 @@
 ) WHERE 1 = 1 {parsed_date_from} {parsed_date_to}
 """
 
-AGGREGATE_SQL = """
+ACTIVE_USERS_AGGREGATE_SQL = """
+SELECT
+    {aggregate_operation} AS total
+FROM ({event_query}) events
+WHERE 1 = 1 {parsed_date_from_prev_range} - INTERVAL {prev_interval} {parsed_date_to}
```

Comment:
Note: We could replace sql interpolation here via `toIntervalDays(%(interval_days)s)`

Reply:
We definitely could, but `prev_interval` is an existing theme, so I'll keep it in this PR